src/ethereum/rfc003/mod.rs

```diff
@@ -3,3 +3,15 @@ pub mod ether_htlc;
 
 pub use erc20_htlc::Erc20Htlc;
 pub use ether_htlc::EtherHtlc;
+
+// ascii of "redeemed"
```

Comment:
Small nit: can we have these as nice documentation with rustdoc please?

Also, I am not quite sure "ascii of redeemed" is correct? The word redeemed _is_ already ASCII, no?
This looks more like hex-encoded ASCII codepoints to me.

```
/// The log message emitted when the HTLC is redeemed.
///
/// These are the hex-encoded ASCII-codepoints of the word "redeemed", padded to a length of 64 characters.
pub const REDEEMED_LOG_MSG: &str =
    "72656465656d6564000000000000000000000000000000000000000000000000";
```

Reply:
Good point.